pydantic_ai_slim/pydantic_ai/_cli/_web.py

```diff
@@ -0,0 +1,146 @@
+"""CLI command for launching a web chat UI for agents."""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from typing import Any
+
+from pydantic import ImportString, TypeAdapter, ValidationError
+from rich.console import Console
+
+from pydantic_ai import Agent
+from pydantic_ai.builtin_tools import AbstractBuiltinTool, get_builtin_tool_cls
+from pydantic_ai.mcp import MCPServerSSE, MCPServerStdio, MCPServerStreamableHTTP, load_mcp_servers
+from pydantic_ai.models import infer_model
+from pydantic_ai.ui._web import create_web_app
+
+__all__ = ['_run_web_command']
```

Comment:
Remove this, it's already outdated as this whole module is private, and we use other functions than this one like _load_agent.